stable/polaris/templates/dashboard.deployment.yaml

```diff
@@ -112,9 +112,13 @@ spec:
       {{- end }}
       serviceAccountName: {{ template "polaris.serviceAccountName" . }}
       nodeSelector:
+      {{- if .Values.dashboard.nodeSelector }}
```

Comment:
@mac2000 it's probably better to change this default in the `values.yaml`. That way it can still be unset

Reply:
maybe, to be fair I'm not sure, the initial intent was to make sure this change won't brake any existing installation, but indeed if that's not the case then a change to default values may be made instead